src/macros.rs

```diff
@@ -18,8 +18,27 @@ macro_rules! hook {
     };
 }
 
+#[macro_export]
+macro_rules! status_set {
+    ($status_type:ident $message:expr) => {
+        $crate::status_set(
+            $crate::Status {
+                status_type: $crate::StatusType::$status_type,
+                message: $message.to_string()
+            }
+        )
+    }
+}
+
 #[cfg(test)]
-mod tests {
+mod tests{
+    #[allow(dead_code)]
+    mod status_set {
+        fn it_compiles_correctly() {
+            let _ = status_set!(Maintenance "Doing stuff");
```

Comment:
Would you rather have the macro swallow the result: ie: the ```let _ =``` could be inside of the macro